### PR TITLE
fix(icons): migrate conversation avatars to MDI icons

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Move avatar icons to MDI appearance
* Fix contrast for the light theme

#### 🖼️ Making the light theme contrast
`#999999` and `#6B6B6B` aligned with `var(--color-text-maxcontrast)`

| 🏚️ Before | 🏡 After |
|------------|----------|
| image | image | 
image | image | 

#### 🖼️ Switch to material design icons

| 🏚️ Before | 🏡 After | 🏚️ Before | 🏡 After |
|------------|----------|-|-|
| Group | Group | Public | Public |
| image | image | image | image |
| Mail | Mail | Password | Password |
| image | image | image | image |
| Phone | Phone | File | File |
| image | image | image | image |
| User | User |
| image | image |

### 🚧 Follow-up

- [ ] Replace public icon with group icon -> add  "status-like" icon for indicating public conversations

### 🏁 Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [x] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: img/icon-conversation-user-bright.svg [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" height="16" width="16" version="1.1">
-	<circle r="16" cx="16" cy="16" fill="#DBDBDB"/>
-	<g transform="translate(8.000000, 8.000000)">
-		<path fill="#fff" d="m5 3.8c0 1.4 0.1 2.4 0.8 3.5 0.2 0.286 0.5 0.35 0.7 0.6 0.135 0.5 0.24 0.98 0.1 1.5-1.275 0.45-2.49 1-3.6 1.6-0.85 0.6-0.785 0.31-1 2.3-0.16 1.59 3.5 1.7 6 1.7s6.163-0.1 6-1.7c-0.215-2-0.23-1.71-1-2.3-1.1-0.654-2.45-1.167-3.6-1.6-0.15-0.56-0.04-0.973 0.1-1.5 0.235-0.25 0.5-0.363 0.7-0.6 0.69-0.885 0.8-2.425 0.8-3.5 0-1.59-1.43-2.8-3-2.8-1.75 0-3 1.43-3 2.8z"/>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 44 44" height="22" width="22" version="1.1">
+	<circle r="22" cx="22" cy="22" fill="#DBDBDB"/>
+	<g transform="translate(10.000000, 10.000000)">
+		<path fill="#000" d="M12,4A4,4 0 0,1 16,8A4,4 0 0,1 12,12A4,4 0 0,1 8,8A4,4 0 0,1 12,4M12,14C16.42,14 20,15.79 20,18V20H4V18C4,15.79 7.58,14 12,14Z"/>
```
[the remark this answers is not in the document]
Comment (nickvergessen) [reply img/icon-conversation-user-bright.svg:4]: But the background color seems to have changed in the server. So adjusting that should restore the contrast as well?

[the remark this answers is not in the document]
Comment (Antreesy) [reply img/icon-conversation-user-bright.svg:4]: Isn't there an additional images, like 'user-white' for that purpose, without background?

[the remark this answers is not in the document]
Comment (Antreesy) [reply img/icon-conversation-user-bright.svg:4]: Shall we adjust for these colors then?
Theme | Dark | Light
--|--|--
Image | image | image
fill | `#000000` | `#ffffff`
background-color | var(--color-text-maxcontrast); `#999999` | var(--color-text-maxcontrast); `#6b6b6b`
In Talk | image | image

**Review by szaimen — APPROVED** [review]
New icons and colors look good to me now from design perspective :)
(didnt review the code and didnt test)

**Review by DorraJaouad — APPROVED** [review]

File: img/icon-conversation-group-dark.svg [hunk 2]
```diff
@@ -1,6 +1,6 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" height="16" width="16" version="1.1">
-	<circle r="16" cx="16" cy="16" fill="#3B3B3B"/>
-	<g transform="translate(8.000000, 8.000000)">
-		<path fill="#fff" d="m9 1c-1.746 0-3 1.43-3 2.8 0 1.4 0.1 2.4 0.8 3.5 0.224 0.29 0.485 0.35 0.7 0.6 0.135 0.5 0.24 1 0.1 1.5-0.436 0.153-0.85 0.332-1.27 0.5-0.51-0.273-1.1-0.5-1.61-0.7-0.07-0.28-0.02-0.487 0.05-0.75 0.12-0.125 0.23-0.18 0.36-0.3 0.37-0.45 0.39-1.21 0.39-1.75 0-0.8-0.72-1.4-1.5-1.4-0.87 0-1.5 0.72-1.5 1.4h-0.02c0 0.7 0.05 1.2 0.4 1.75 0.1 0.15 0.242 0.175 0.35 0.3 0.0674 0.25 0.121 0.5 0.05 0.75-0.64 0.223-1.244 0.5-1.8 0.8-0.42 0.3-0.233 0.182-0.5 1.15-0.124 0.5 1.3 0.73 2.32 0.81-0.05 0.275-0.12 0.64-0.32 1.34-0.32 1.25 4.353 1.7 6 1.7 2.43 0 6.313-0.456 5.98-1.7-0.52-1.94-0.208-1.71-0.98-2.3-1.09-0.654-2.452-1.167-3.6-1.6-0.15-0.557-0.04-0.97 0.1-1.5 0.235-0.25 0.5-0.36 0.72-0.6 0.69-0.884 0.78-2.424 0.78-3.5 0-1.586-1.43-2.8-3-2.8z"/>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 44 44" height="22" width="22" version="1.1">
+	<circle r="22" cx="22" cy="22" fill="#999999"/> // aligned with var(--color-text-maxcontrast)
```
Comment (nickvergessen) [inline img/icon-conversation-group-dark.svg:2]: Not valid XML btw with the comment

File: img/icon-conversation-mail-bright.svg [hunk 3]
```diff
@@ -1,6 +1,6 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" height="16" width="16" version="1.1">
-	<circle r="16" cx="16" cy="16" fill="#DBDBDB"/>
-	<g transform="translate(8.000000, 8.000000)">
-		<path fill="#fff" d="m0.89 3c-0.4924 0-0.89 0.4-0.89 0.89v8.2202c0 0.493 0.4 0.89 0.89 0.89h14.22c0.492 0 0.89-0.4 0.89-0.89v-8.2202c0-0.4924-0.4-0.89-0.89-0.89zm0.75 1.0278 6.0827 6.0817h0.52773l6.1102-6.0817 0.611 0.6109-3.6384 3.6934 2.75 2.8047-0.61102 0.61092-2.8052-2.8047-2.0275 2.0549h-1.2776l-2.0271-2.0553-2.8053 2.8323-0.6111-0.639 2.7774-2.8046-3.666-3.6932z"/>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 44 44" height="22" width="22" version="1.1">
+	<circle r="22" cx="22" cy="22" fill="#6B6B6B"/> // aligned with var(--color-text-maxcontrast)
```
Comment (nickvergessen) [inline img/icon-conversation-mail-bright.svg:2]: also on the bright images

File: img/icon-conversation-group-dark.svg (hunk 2 above)
Comment (Antreesy) [reply img/icon-conversation-group-dark.svg:2]: Will remove then. Should be trackable with this commit anyway

**Review by jancborchardt — COMMENTED** [review]
@Antreesy good icon changes! One thing – after trying it out a bit more, it feels like the color scheme for the placeholders would look better the other way around between dark and light mode – with the new and better contrasting colors of course:
- Light mode: Light background and dark icon
- Dark mode: Dark background and light icon (as it was before)

Or was it switched around for specific reasons?

**Comment by Antreesy** [comment]
> Or was it switched around for specific reasons?

Was aligned with colors used at server apps (Files):
image
image

**Comment by jancborchardt** [comment]
Ah right, and that was also for improved accessibility. All good then and thanks for the note @Antreesy! :)

**Pull request merged**